airflow/executors/kubernetes_executor.py

```diff
@@ -773,10 +777,16 @@ def sync(self) -> None:
             self.log.debug('self.running: %s', self.running)
         if self.queued_tasks:
             self.log.debug('self.queued: %s', self.queued_tasks)
-        assert self.kube_scheduler, NOT_STARTED_MESSAGE
-        assert self.kube_config, NOT_STARTED_MESSAGE
-        assert self.result_queue, NOT_STARTED_MESSAGE
-        assert self.task_queue, NOT_STARTED_MESSAGE
+        if not self.worker_uuid:
+            raise AirflowException(NOT_STARTED_MESSAGE)
+        if not self.kube_scheduler:
+            raise AirflowException(NOT_STARTED_MESSAGE)
+        if not self.kube_config:
+            raise AirflowException(NOT_STARTED_MESSAGE)
+        if not self.result_queue:
+            raise AirflowException(NOT_STARTED_MESSAGE)
+        if not self.task_queue:
```

Comment:
Can we change this to `all([self.worker_uuid, self.kube_config, ...])`? The message is the same.

Reply:
But then you do not know which of them was null. If you use "all" you won't figure it out from the stack trace.

Reply:
BTW. This was the main reason why I thought asserts are much better in this case - because I wanted to keep one line per check.

Reply:
Are you ok @nuclearpinguin  with leaving this in?